lib/elixir/lib/calendar/naive_datetime.ex

```diff
@@ -229,12 +274,17 @@ defmodule NaiveDateTime do
       63579428950
 
   """
-  @spec diff(t, t, System.time_unit) :: integer
-  def diff(%NaiveDateTime{} = naive_datetime1,
-           %NaiveDateTime{} = naive_datetime2,
-           unit \\ :second) do
+  @spec diff(t, t, System.time_unit()) :: integer
+  def diff(
+        %NaiveDateTime{} = naive_datetime1,
+        %NaiveDateTime{} = naive_datetime2,
+        unit \\ :second
+      ) do
     if not Calendar.compatible_calendars?(naive_datetime1.calendar, naive_datetime2.calendar) do
-      raise ArgumentError, "cannot calculate the difference between #{inspect naive_datetime1} and #{inspect naive_datetime2} because their calendars are not compatible and thus the result would be ambiguous"
+      raise ArgumentError,
+            "cannot calculate the difference between #{inspect(naive_datetime1)} and #{
+              inspect(naive_datetime2)
+            } because their calendars are not compatible and thus the result would be ambiguous"
```

Comment:
this break feels a little strange to me, but the only way I could come up to avoid it was to concatenate multiple strings, which feels worse. any comments?

Reply:
Please do split the message into multiple strings and concatenate them with `<>`. It's what we do all over the codebase 👍

Reply:
Done! Thank you for reviewing it!